hi-ml-histopathology/src/histopathology/configs/classification/custom_subset/panda/sub_train_slides.csv

```diff
@@ -0,0 +1,90 @@
+d8888c68d56232a148462a0096b4579f,karolinska,4,4+4
+d89f2be0f1683b034375f80d446e2407,karolinska,0,0+0
+d8a2411e48aa5feb4fa2ef30d0d7b4b1,karolinska,0,0+0
+d8aa8df17952aa8028bcb040964989c0,karolinska,1,3+3
+d8abf403ab5663c07528475647e0db5a,radboud,1,3+3
+d8aef66ed061a62078ef30ab96bebe01,karolinska,2,3+4
+d8b48c0404394e83af61c15ad7eea1a3,radboud,1,3+3
+d8b611006a6b9a4dfd31032107923951,radboud,4,4+4
+d8b8316a7e615a79e9d725e9f235918f,karolinska,4,4+4
+d8c4a0a75d72a1f8e94b0939fe27855b,karolinska,0,0+0
+d8c9f66055f7d4efda021bee79e71a9c,karolinska,1,3+3
+d8dab9eef691dc2a83874a3560fbaddb,radboud,1,3+3
+d8dc5358795768819f229a681d2dc483,karolinska,1,3+3
+d8e6c24dadbb377b6934a21ba7ce02c4,radboud,1,3+3
+d8f6b0abbcf5d89fc684cdfa77f0a655,karolinska,1,3+3
+d8f6b2a76413e5b03d0ee89d2792e610,radboud,5,5+4
+d8fc4bf3dec4c8dfbc366e3d7512c602,karolinska,1,3+3
+d908ac781e507048dca9b45d93fcec44,karolinska,3,4+3
+d90f5db28f3ff7c708e3c95d9596e26f,karolinska,1,3+3
+d91b799dcfec74a8fcb96a53f2022274,karolinska,0,0+0
+d92fc82bea7ec08b5d7bac547848abf3,karolinska,0,0+0
+d931f2fc50aca040ea3fd6e917666483,karolinska,1,3+3
+d936397deeda339f9e7ecb5c81c2250a,karolinska,2,3+4
+d94046cd69c8102ff1e1a871077c7984,radboud,5,4+5
+d9471d56985f343c8f300d622628ffaf,karolinska,1,3+3
+d960f3c9327c1baa21ed849d867e29a1,radboud,4,4+4
+d9652aa1aed6ef06e2c1f8181b734d9e,radboud,0,negative
+d97f89398bb6e6e441de40a85a55f762,karolinska,5,4+5
+d9872b11bb4b59a34062b36e8a64e3a7,karolinska,5,5+5
+d98d4b443eaaad06c3244e059b34053f,radboud,4,4+4
+d98de7b36e1362dfa74903ec96e53c15,radboud,0,negative
+d996d408cbc9200c7254ab6955e31959,radboud,2,3+4
+d997bd54e6a545c95817a62b77fb43df,radboud,3,4+3
+d9a122f3cfcf06a7768de8f5eb27a6d1,karolinska,0,0+0
+d9a927bfb26b22cde6b52515c9adda7b,radboud,1,3+3
+d9abfe7bab306ba7777c4f8e619957ba,karolinska,0,0+0
+d9ad3f3ed67f14c9c9b35a5c4e85400b,radboud,2,3+4
+d9b1722876190bd66e46ff2cec3847d6,radboud,1,3+3
+d9b76603ea5a351942248655981b90b0,radboud,2,3+4
+d9ccb1baa42354825862d4e4b3e48507,karolinska,0,0+0
+d9d9d64870b78f0bd2be305777f95e52,radboud,1,3+3
+d9e73507e94ba12a7c566087dcf8348f,karolinska,1,3+3
+d9ef8f2b1620b5accb8fd81fbc873b39,karolinska,2,3+4
+d9fab0dbd3658bb592b14d07a3c167e0,karolinska,0,0+0
+da02528d2ac0b705067ad134179791d2,radboud,5,5+4
+da077ff5258dfb7cd6d604d995de7619,karolinska,0,0+0
+da07bf2ac3a36b2ff58e35561f15613f,radboud,0,negative
+da09fe1419dcf0c2e16c0b2a43979c43,karolinska,2,3+4
+da0c4c33709fb5d3372c7e0685b6a767,karolinska,1,3+3
+da15226ed2a9ca5a83e831a9c57f5380,radboud,0,negative
+da1bf67eeb0d10c8021d69a0d6128f15,radboud,3,4+3
+da1de69a3097c5cecfc5a5c20f125340,radboud,0,negative
+da2319b5a8f33c13bcf2380c25b78e37,radboud,3,4+3
+da2502ee900c4fd5653d4433d05c8cde,radboud,2,3+4
+da4b2e029abb2741bcfef60fa7ab10b0,radboud,5,5+4
+da517beeeaf103a75e2a716d46643b49,radboud,5,5+4
+da5bbea85af39c137a264ee0bd56ef50,radboud,5,4+5
+da5e1279af2669d8b8abf40d450bd896,karolinska,0,0+0
+da5fc79f09ea997183453a16d8f3c393,karolinska,0,0+0
+da610fb22e9ba742ebb7a9117b2f24cc,radboud,2,3+4
```

Comment:
We selected 89 slides for training and 14 for validation. Just checking these numbers were what you wanted? I thought we were thinking much smaller numbers for benchmarking at the beginning, but I might just have lost part of the conversation

Reply:
Yes, indeed! I just run into some issues and had to add more examples to check my assumptions. I will reduce that to the numbers we agreed upon.